kotest-extensions/kotest-extensions-allure/src/jvmMain/kotlin/io/kotest/extensions/allure/AllureWriter.kt

```diff
@@ -121,3 +122,12 @@ fun TestCase.story(): Label? = this.spec::class.findAnnotation<Story>()?.let { R
 fun TestCase.owner(): Label? = this.spec::class.findAnnotation<Owner>()?.let { ResultsUtils.createOwnerLabel(it.value) }
 fun TestCase.issue() = spec::class.findAnnotation<Issue>()?.let { ResultsUtils.createIssueLink(it.value) }
 fun TestCase.description() = spec::class.findAnnotation<io.qameta.allure.Description>()?.value
+
+fun String.convertToSeverity(): SeverityLevel? = when (this) {
+   "MINOR" -> SeverityLevel.MINOR
```

Comment:
One more change required - Trivial should be the lowest level to match Allure.

Reply:
May be i don't understand - 
Now    TRIVIAL(0);
and i changed the order of the condition :
   "BLOCKER" -> SeverityLevel.BLOCKER
   "CRITICAL" -> SeverityLevel.CRITICAL
   "NORMAL" -> SeverityLevel.NORMAL
   "MINOR" -> SeverityLevel.MINOR
   "TRIVIAL" -> SeverityLevel.TRIVIAL

Reply:
@sksamuel or you about something else?